tests/lib/Comments/ManagerTest.php

```diff
@@ -701,6 +703,51 @@ public function testDeleteCommentsAtObject() {
 		$this->assertTrue($wasSuccessful);
 	}
 
+	public function testDeleteMessageExpiredAtObject(): void {
+		$ids = [];
+		$ids[] = $this->addDatabaseEntry(0, 0, null, null, null, new \DateTime('+2 hours'));
+		$ids[] = $this->addDatabaseEntry(0, 0, null, null, null, new \DateTime('+2 hours'));
+		$ids[] = $this->addDatabaseEntry(0, 0, null, null, null, new \DateTime('+2 hours'));
+		$ids[] = $this->addDatabaseEntry(0, 0, null, null, null, new \DateTime('-2 hours'));
+		$ids[] = $this->addDatabaseEntry(0, 0, null, null, null, new \DateTime('-2 hours'));
+		$ids[] = $this->addDatabaseEntry(0, 0, null, null, null, new \DateTime('-2 hours'));
+
+		$manager = new Manager(
+			$this->connection,
+			$this->createMock(LoggerInterface::class),
+			$this->createMock(IConfig::class),
+			Server::get(ITimeFactory::class),
+			new EmojiHelper($this->connection),
+			$this->createMock(IInitialStateService::class)
+		);
+
+		// just to make sure they are really set, with correct actor data
+		$comment = $manager->get(strval($ids[1]));
```

Comment:
```suggestion
		$comment = $manager->get((string)$ids[1]);
```

micro optimization :)

Reply:
I did other test to check the opcode in site 3v4l.org using the follow code:

```php
echo (string) 1;
echo strval(1);
```
...and the opcode of second row have one more step, a return:

```
line      #* E I O op                           fetch          ext  return  operands
-------------------------------------------------------------------------------------
    3     0  E >   CAST                                          6  ~0      1
          1        ECHO                                                     ~0
    4     2        CAST                                          6  ~1      1
          3        ECHO                                                     ~1
          4      > RETURN                                                   1
```

https://3v4l.org/ZGBcX/vld

Reply:
Ok, it was intval vs int-casting. Never mind then!